build/scripts/tests.sh

```diff
@@ -20,43 +20,38 @@ TARGET_FRAMEWORK=netcoreapp2.0
 XUNIT_CONSOLE_VERSION=$(get_package_version dotnet-xunit)
 XUNIT_CONSOLE=${NUGET_DIR}/dotnet-xunit/${XUNIT_CONSOLE_VERSION}/tools/${TARGET_FRAMEWORK}/xunit.console.dll
 
-echo Using $XUNIT_CONSOLE
+echo "Using ${XUNIT_CONSOLE}"
 
 # Need to publish projects that have runtime assets before running tests
 NEED_PUBLISH=(
-    'src\Compilers\CSharp\Test\Symbol\CSharpCompilerSymbolTest.csproj'
+    'src/Compilers/CSharp/Test/Symbol/CSharpCompilerSymbolTest.csproj'
 )
-for p in $NEED_PUBLISH
+
+for project in $NEED_PUBLISH
 do
-    echo Publishing ${p}
-    dotnet publish --no-restore ${SRC_PATH}/${p} -p:RuntimeIdentifier=${RUNTIME_ID} -p:TargetFramework=${TARGET_FRAMEWORK} -p:SelfContained=true
+    echo "Publishing ${project}"
+    dotnet publish --no-restore ${ROOT_PATH}/${project} -r ${RUNTIME_ID} -f ${TARGET_FRAMEWORK} -p:SelfContained=true
 done
 
 # Discover and run the tests
-mkdir -p ${LOG_DIR}
-pushd ${UNITTEST_DIR}
+mkdir -p "${LOG_DIR}"
 
-for d in *
+for TEST_PATH in ${UNITTEST_DIR}/*/${TARGET_FRAMEWORK}
```

Comment:
this loop variable is upper case but the one above is lower case. think we should pick a style here

Reply:
Agreed. I was keeping existing style of the variable already defined (the line I deleted just below this). [A random google shows that most vars should be lowercase](https://stackoverflow.com/a/673940).